vispy/color/colormap.py

```diff
@@ -508,7 +510,158 @@ def map(self, t):
+        The number indicating the hue value, which refers to a "true" color,
+        without any shading or tinting. Must be in the range [0, 360]. Defaults
+        to 200 (blue).
+    saturation_range : array-like, optional
+        The saturation represents how "pure" a color is. Less saturation means
+        more white light mixed in the color. A fully saturated color means
+        the pure color defined by the hue. No saturation means completely
+         hite. This colormap changes the saturation, and with this parameter
+        you can specify the lower and upper bound. Default is [0.2, 0.8].
+    value : scalar, optional
+        The number represting the value component of a color. This defines the
+        "brightness" of a color: a value of 0.0 means completely black while a
+        value of 1.0 means the color defined by the hue without shading. Must
+        be in the range [0, 1.0]. The default value is 1.0.
+
+    Notes
+    -----
+    For more information about the hue values see the `wikipedia page`_.
+
+    .. _wikipedia page: https://en.wikipedia.org/wiki/Hue
+    """
+
+    def __init__(self, hue=200, saturation_range=[0.1, 0.8], value=1.0):
+        colors = ColorArray([
+            (hue, saturation_range[0], value),
+            (hue, saturation_range[1], value)
+        ], color_space='hsv')
+        super(_SingleHue, self).__init__(colors)
+
+
+class _HSL(Colormap):
+    """A colormap which is defined by n evenly spaced points in
+    a circular color space.
+
+    This means that we change the hue value while keeping the
+    saturation and value constant.
+
+    Parameters
+    ---------
+    n_colors : int, optional
+        The number of colors to generate.
+    hue_start : int, optional
+        The hue start value. Must be in the range [0, 360], the default is 0.
+    saturation : float, optional
+        The saturation component of the colors to generate. The default is
+        fully saturated (1.0). Must be in the range [0, 1.0].
+    value : float, optional
+        The value component of the colors to generate or "brightness". Must
+        be in the range [0, 1.0], and the default is 1.0
+    controls : array-like, optional
+        The list of control points for the colors to generate. It should be
+        an increasing list of floating-point number between 0.0 and 1.0.
+        The first control point must be 0.0. The last control point must be
+        1.0. The number of control points depends on the interpolation scheme.
+    interpolation : str, optional
+        The interpolation mode of the colormap. Default: 'linear'. Can also
+        be 'zero'.
+        If 'linear', ncontrols = ncolors (one color per control point).
+        If 'zero', ncontrols = ncolors+1 (one color per bin).
+    """
+
+    def __init__(self, ncolors=6, hue_start=0, saturation=1.0, value=1.0,
+                 controls=None, interpolation='linear'):
+        hues = np.linspace(0, 360, ncolors + 1, dtype='int')[:-1]
```

Comment:
Things like this usually depend on version, looking at the docs you can see it was introduced in 1.9:

http://docs.scipy.org/doc/numpy/reference/generated/numpy.linspace.html

So safer to just use `.astype(int)`